core/src/main/java/io/github/tcdl/msb/impl/ResponderServerImpl.java

```diff
@@ -47,7 +47,7 @@ static ResponderServerImpl create(String namespace,  MessageTemplate messageTemp
     public ResponderServer listen() {
         ChannelManager channelManager = msbContext.getChannelManager();
 
-        channelManager.subscribe(namespace,
+        boolean isAlreadySubscribed = channelManager.subscribe(namespace,
```

Comment:
>Why not throw exception on the lower level in ChannelManager? 
Because we will receive that Exception in method publish(RequesterImpl). It  will break functionality(more then one Collectors per topic) which we achieved with help of CollectorManager)